tests/modules/encounters/resources/utils.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+Project resources utils
+-------------
+"""
+import json
+from tests import utils as test_utils
+
+PATH = '/api/v1/encounters/'
+
+
+def create_encounter(flask_app_client, user):
```

Comment:
This looks familiar :) 
Any reason you've chosen to have the expected status code and content validation in delete but not in create and read. I would have thought it would make the tests simpler if this functionality was in all of the helpers